worker.py

```diff
@@ -0,0 +1,140 @@
+from email.mime import text
+import logging
+import pika
+import smtplib
+from sqlalchemy.orm import exc
+
+from mailbbgun import app
+from mailbbgun import models
+
+
+if app.config['DEBUG']:
+    logging.basicConfig(level=logging.DEBUG)
+
+
+_BANNER = """
+  __  __       _ _   ____  ____     _____
+ |  \/  |     (_) | |  _ \|  _ \   / ____|
+ | \  / | __ _ _| | | |_) | |_) | | |  __ _   _ _ __
+ | |\/| |/ _` | | | |  _ <|  _ <  | | |_ | | | | '_ \\
+ | |  | | (_| | | | | |_) | |_) | | |__| | |_| | | | |
+ |_|  |_|\__,_|_|_| |____/|____/   \_____|\__,_|_| |_|
+
+        _  ____________.---------.
+        \`'  __________|_________| o o o
+        /   (_)__]
+       |    |
+      .'   .'
+      |____]
+"""
+_DELIVERY_MODE_PERSISTENT = 2
+_LOG = logging.getLogger(__name__)
+
+
+class MailBBGunWorker():
+
+    def configure_rabbitmq(self):
+        self.connection = pika.BlockingConnection(
+            pika.ConnectionParameters(app.config['RABBITMQ_HOST'])
+        )
+        self.channel = self.connection.channel()
+        self.channel.queue_declare(queue='mailbbgun', durable=True)
+        self.channel.queue_bind(exchange='amq.direct', queue='mailbbgun')
+
+        self.retry_channel = self.connection.channel()
+        self.retry_channel.queue_declare(
+            queue='retry_delay',
+            durable=True,
+            arguments={
+                'x-message-ttl': app.config['RETRY_DELAY_MS'],
+                'x-dead-letter-exchange': 'amq.direct',
+                'x-dead-letter-routing-key': 'mailbbgun'
+            }
+        )
+
+    def process_message(self, ch, method, properties, body):
+        message_id = body.decode('UTF-8')
```

Comment:
Rabbit is waiting for an ack from the worker.  If the worker restarts here, rabbit will notice the connection drop and resend the message to another worker.